extensions-core/caffeine-cache/README.md

```diff
@@ -0,0 +1,41 @@
+Druid Caffeine Cache
+--------------------
+
+A local cache implementation for Druid based on [Caffeine](https://github.com/ben-manes/caffeine). Requires a JRE with a fix for https://bugs.openjdk.java.net/browse/JDK-8078490
+
+# Versioning
+
+The versioning works like this: `druid_version.patch_set`. Such that `druid_version` is the version of druid the extension was compiled against, and `patch_set` is the "version" of the extension.
+
+# How to use
+The maven artifact coordinate for this extension is `io.druid.extensions:druid-caffeine-cache`
+
+For Druid 0.9.0 and later, the extension can be included by pulling the jars using the `pull-deps` tool, and including the extension directory in the extension load list.
+
+### Jars
+
+For the sake of sanity if you are manually making an extension directory, release requires the following jars: 
```

Comment:
I can edit it to be clearer. I was trying to indicate this extension should be packaged with two jars (as of 0.9.0)